src/index.js

```diff
@@ -1 +1,90 @@
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+const server = http.createServer((req, res) => {
+  if (req.url === '/') {
+    const pathFileHTML = path.join(__dirname, 'pages', 'index.html');
+
+    const readStream = fs.createReadStream(pathFileHTML);
+
+    res.statusCode = 200;
+    res.setHeader('Content-Type', 'text/html');
+
+    readStream.pipe(res);
+
+    readStream.on('error', (err) => {
+      res.statusCode = 404;
+      res.end('Error: ' + err);
+    });
+
+    return;
+  }
+
+  if (req.url === '/data') {
+    const multiparty = require('multiparty');
```

Comment:
Consider moving the 'require' statement for 'multiparty' to the top of the file.